packages/@ngtools/webpack/src/transformers/replace_bootstrap.ts

```diff
@@ -0,0 +1,114 @@
+import * as ts from 'typescript';
```

Comment:
Unittests for this file?  It's quite sensitive code that I'd like to avoid regressions.